firebase-installations/src/main/java/com/google/firebase/installations/CrossProcessLock.java

```diff
@@ -25,67 +26,40 @@
 /** Use file locking to acquire a lock that will also block other processes. */
 class CrossProcessLock {
   private static final String TAG = "CrossProcessLock";
-  private final FileChannel channel;
-  private final FileLock lock;
 
-  private CrossProcessLock(FileChannel channel, FileLock lock) {
-    this.channel = channel;
-    this.lock = lock;
+  interface CrossProcessPersistedInstallation {
+    PersistedInstallationEntry getPersistedInstallations();
   }
 
   /**
-   * Create and return a lock that is exclusive across processes. If another process has the lock
-   * then this call will block until it is released.
+   * Create a lock that is exclusive across processes. If another process has the lock then this
+   * call will block until it is released.
```

Comment:
Will there be a time out?
Let's say we only wait on this lock for 1 or 5s?
If we are waiting on a file lock, then 1s should be enough right? ... but even 2, 3, or 5 should be fine.

Reply:
@andirayo What is the intent of timing out? is if acceptable to proceed if acquiring the lock fails? if so then what is the value of the lock in the first place?

Reply:
I agree, there is no need to timeout here.